### DISCUSSION TITLE
( RUN WINDOWS 11 VM) On SCHOOL CHROME------ Found it from the bypass hub

### DISCUSSION THREAD

**Discussion by Luigi-Master** [3kh0/ext-remover#1227] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
steps to run windows 11 on your chrome now, (also lets you access blocked sites.)

1. copy and paste this link on a new tab ----https://learn.microsoft.com/en-us/training/modules/implement-common-integration-features-finance-ops/10-exercise-1 

2. Press Launch VM mode and sign in with your existing Microsoft account or create a new account. 

3. If you were not signed in previously it will take you back to the same page so just press launch VM mode again.

4. Wait for the build setup and for it to launch

5. Now once you get into the sign in screen, enter this password,----  pass@word1 

6. You should be in now and on the home screen, (*you can also run exe files too)  

Have Fun!

**Comment by buddyted** [comment, marked as answer]
this is just "Microsoft Labs Virtualbox but it got 20 doses of the COVID Vaccine"

**Reply by Luigi-Master** [reply]
well you can also look at it as an unblocker site

**Reply by buddyted** [reply]
Fair atleast this is unblocked
